games-emulation/higan/higan-116_pre.ebuild

```diff
@@ -0,0 +1,164 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop toolchain-funcs xdg
+
+DESCRIPTION="A multi-system emulator focused on accuracy, preservation, and configurability"
+HOMEPAGE="https://github.com/higan-emu/higan"
+SRC_URI="https://github.com/higan-emu/${PN}/archive/nightly.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="
+	alsa ao +byuu +core-cv +core-fc +core-gb +core-gba +core-md +core-ms
+	+core-msx +core-n64 +core-ngp +core-pce +core-ps1 +core-sfc +core-sg
+	+core-ws +gtk higan-ui icarus openal +opengl oss +pulseaudio qt5 +sdl udev
```

Comment:
Please reconsider adding all those `core-*` flags.
- They do not pull in any additional dependency
- What is the net benefit (in size) of disabling individual cores? If it is not substantial by modern standards, why bother?
- Is there any practical gain (besides install size) from being able to disable one or several cores?

Reply:
I suppose most users of higan are using it specifically because it supports multiple cores. So perhaps there's not much benefit afterall in providing these USE flags because I imagine most users would just leave them all enabled anyway. I think it's best to just remove the USE flags then and just build all cores unconditionally.